[2024-06-18T13:21:43.863+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T13:21:43.882+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_mysql_analytics.delete_data_from_table manual__2024-06-18T13:21:41.855965+00:00 [queued]>
[2024-06-18T13:21:43.889+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_mysql_analytics.delete_data_from_table manual__2024-06-18T13:21:41.855965+00:00 [queued]>
[2024-06-18T13:21:43.889+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-06-18T13:21:43.898+0000] {taskinstance.py:2330} INFO - Executing <Task(MySqlOperator): delete_data_from_table> on 2024-06-18 13:21:41.855965+00:00
[2024-06-18T13:21:43.903+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6290) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T13:21:43.904+0000] {standard_task_runner.py:63} INFO - Started process 6291 to run task
[2024-06-18T13:21:43.904+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_mysql_analytics', 'delete_data_from_table', 'manual__2024-06-18T13:21:41.855965+00:00', '--job-id', '747', '--raw', '--subdir', 'DAGS_FOLDER/mysql-analytics-test_dag.py', '--cfg-path', '/tmp/tmpc_u3p9zd']
[2024-06-18T13:21:43.904+0000] {standard_task_runner.py:91} INFO - Job 747: Subtask delete_data_from_table
[2024-06-18T13:21:43.934+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_mysql_analytics.delete_data_from_table manual__2024-06-18T13:21:41.855965+00:00 [running]> on host 195daacf7793
[2024-06-18T13:21:43.989+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_mysql_analytics' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T13:21:41.855965+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T13:21:41.855965+00:00'
[2024-06-18T13:21:43.989+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T13:21:43.998+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2024-06-18' and dag_id = 'dag_with_mysql_analytics';
        
[2024-06-18T13:21:44.004+0000] {base.py:84} INFO - Using connection ID 'analytics_localhost' for task execution.
[2024-06-18T13:21:44.008+0000] {base.py:84} INFO - Using connection ID 'analytics_localhost' for task execution.
[2024-06-18T13:21:44.013+0000] {sql.py:487} INFO - Running statement: 
            delete from dag_runs where dt = '2024-06-18' and dag_id = 'dag_with_mysql_analytics';
        , parameters: None
[2024-06-18T13:21:44.017+0000] {sql.py:496} INFO - Rows affected: 0
[2024-06-18T13:21:44.018+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T13:21:44.024+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_mysql_analytics, task_id=delete_data_from_table, run_id=manual__2024-06-18T13:21:41.855965+00:00, execution_date=20240618T132141, start_date=20240618T132143, end_date=20240618T132144
[2024-06-18T13:21:44.077+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T13:21:44.085+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297: AirflowProviderDeprecationWarning: Call to deprecated class MySqlOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-06-18T13:21:44.093+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-18T13:21:44.095+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
